student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
```

Comment:
- [ ] Hello! When setting up a classroom for a student, ensure that the student is also added to the list of students in that classroom. We should implement a method that establishes the 'belongs-to' relationship, indicating that a student belongs to a specific classroom. 📚👩‍🏫

```suggestion
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```